daprdocs/content/en/operations/components/setup-pubsub/supported-pubsub/setup-apache-kafka.md

```diff
@@ -5,57 +5,65 @@ linkTitle: "Apache Kafka"
 description: "Detailed documentation on the Apache Kafka pubsub component"
 ---
 
-## Setup Kafka
-{{< tabs "Self-Hosted" "Kubernetes">}}
-
-{{% codetab %}}
-You can run Kafka locally using [this](https://github.com/wurstmeister/kafka-docker) Docker image.
-To run without Docker, see the getting started guide [here](https://kafka.apache.org/quickstart).
-{{% /codetab %}}
-
-{{% codetab %}}
-To run Kafka on Kubernetes, you can use the [Helm Chart](https://github.com/helm/charts/tree/master/incubator/kafka#installing-the-chart).
-{{% /codetab %}}
-
-{{< /tabs >}}
-
-## Create a Dapr component
-
-The next step is to create a Dapr component for Kafka.
-
-Create the following YAML file named `kafka.yaml`:
+## Format
```

Comment:
I'd update this to "Component format" and add a line underneath saying:

Create a new component file of type `pubsub.redis` and apply it to your Dapr environment using [this guide]({{< ref "howto-publish-subscribe.md#step-2-publish-a-topic" >}}).

Then delete the "Apply the configuration" section